patterns/roks/variables.tf

```diff
@@ -281,6 +282,30 @@ variable "operating_system" {
   }
 }
 
+variable "use_private_endpoint" {
```

Comment:
In SLZ, cluster is private only and ACLs prevent any ingress, so this has to be `true` and should not be an input. I would remove this variable and just hardcode to "true" in config.tf.